src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -263,13 +263,13 @@ private TestCaseParameters GetParametersForTestCase(IMethodInfo method)
                 IParameterInfo[] parameters = method.GetParameters();
                 int argsNeeded = parameters.Length;
                 int argsProvided = Arguments.Length;
-
+                
```

Comment:
I'd rather this wasn't in the git history for this line. :D

Reply:
Oops, good catch. Thanks @jnm2 , I've removed the change here